test/run-tests.py

```diff
@@ -177,11 +177,11 @@ def pytest_pyfunc_call(self, pyfuncitem):
 				"[run]",
 				"omit =",
 			), map(lambda s: "\t" + s, omitted_files),
-			(
-				"[report]",
-				"# Exclude lines specific to some other Python version from coverage",
-				"exclude_lines =",
-			), map(lambda s: "\t" + s, exclusions))))
+				(
```

Comment:
It will behave better here if you move the opening parenthese to the line above. The way it's currently written was actually a workaround to a bug in an older version of the checker. :wink:

Reply:
^ Do this.

Reply:
I had tried simply moving the parenthesis on line 180 up earlier but it didn't like that. The latest change flattens that entire section and passes the linter.